src/functions-and-arrays.js

```diff
@@ -1,13 +1,39 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a
+  };
+  if (a < b) {
+    return b
+  };
+  if (a === b) {
+    return a, b
+  };
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(words) {
+  let longest = ""
+  for (let word of words) {
+    // Return null when array is empty
+    if(!words.length) {
+      return null
+    };
```

Comment:
This should be before the for loop. The for loop will not iterate if the array is empty and so this line will never be reached and the function will return an empty string if the array is empty.